drivers/interrupt_controller/exti_stm32.c

```diff
@@ -334,12 +501,13 @@ static inline void __stm32_exti_isr_22(void *arg)
 	__stm32_exti_isr(22, 23, arg);
 }
 #endif
-#ifdef CONFIG_SOC_SERIES_STM32F7X
+#if defined(CONFIG_SOC_SERIES_STM32F7X) || \
+	defined(CONFIG_SOC_SERIES_STM32MP1X)
 static inline void __stm32_exti_isr_23(void *arg)
 {
 	__stm32_exti_isr(23, 24, arg);
 }
-#endif /* CONFIG_SOC_SERIES_STM32F7X */
+#endif /* CONFIG_SOC_SERIES_STM32{F7X MP1x} */
```

Comment:
We have sanitycheck script that look for non existing Kconfig symbols and we clean them up.
Remove CONFIG_SOC_SERIES_STM32{F7X MP1x}